components/spotify/new-playlists.js

```diff
@@ -0,0 +1,50 @@
+const spotify = require("https://github.com/PipedreamHQ/pipedream/components/spotify/spotify.app.js");
+
+module.exports = {
+  name: "Spotify - New Playlists",
+  description:
+    "Emits an event for each new playlist owned or followed by the current Spotify user.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    spotify,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+
+  async run(event) {
+    let playlists = [];
+    let results;
+    let total = 1;
+    let count = 0;
+    let limit = 20;
+    let offset = 0;
+
+    let params = {
+      limit,
+      offset,
+    };
+
+    while (count < total) {
+      results = await this.spotify.getPlaylists(params);
```

Comment:
Does Spotify allow you to pass a timestamp to the get playlists API endpoint so that, like YouTube, you can filter on playlists created since the last time the source runs?

Again this isn't a huge deal since the deduper will handle duplicate playlists, but just wanted to see.

Reply:
No, the only parameters you can pass to it are limit and offset.